data/remnant missions.txt

```diff
@@ -60,27 +62,94 @@ mission "First Contact: Remnant"
 	
 	on decline
 		event "ember waste label"
+		set "remnant blood test pure"
 		log "Discovered a strange, isolated human community in an otherwise uninhabited region of space south of the Core. They call themselves the Remnant."
 		log "Factions" "Remnant" `Half a millennium ago, during the Alpha Wars, a group of humans fleeing the conflict and looking for a safer place to live discovered a wormhole into a region of space known as the Ember Waste. They built settlements there, and chose to call themselves the Remnant. Although they no longer fear the Alphas as much as they once did, they still have not rejoined human society.`
 			`Since their colonies were first formed, the Remnant's culture has changed drastically. They communicate in sign language, using their voices only for singing. Their food and architecture and much of their technology is strange, and much of it has been borrowed or stolen from nearby alien species.`
 	
 	npc kill
 		government "Remnant"
 		personality heroic coward disables plunders
-		fleet "Large Remnant" 2
+		fleet "Large Remnant"
 
 event "ember waste label"
 	galaxy "label waste"
 		sprite label/waste
 
+mission "Remnant: Second Contact"
+	name "Second chance at First Contact with the Remnant"
+	landing
+	invisible
+	source
+		government "Remnant"
+	description "A Remnant provides you with the opportunity to take a second run at first contact."
```

Comment:
I included those in interests of clarity so if someone gets stumped digging around in their pilot file they would have an explanation for what the mission is. I guess it is a comment, except done in proper data tags instead of just commenting in a line. 

Anyway, is this a problem, or just an informative note that it isn't strictly necessary? I like having it there to make it easier for other people to build on missions that I submit (and I do hope that these inspire others to build missions that branch off them).

Reply:
I can understand having a description for such a reason, but the `name` line is entirely pointless, and I don't see the description as being necessary as it's pretty much just a longer way of stating what the mission title is.

Reply:
OK, I'll take those out.